index.html

```diff
@@ -12,10 +12,11 @@
       }
 
       .container {
```

Comment:
Very nice improvements!

However, this PR does not contain fixes for the results table. Is this intentionally a follow-up or did you miss a commit?

![IMG_B1ABA3E66DBE-1](https://github.com/openHPI/quiz-recap/assets/22977799/bf490feb-a5c1-4987-83a2-586e60b623da)

Reply:
Yes, this will be solved with XI-6348. There we include the logic for truncating each row, and other changes that will improve the results table on mobile view. It was a bit difficult to know which changes would belong in this PR or to the other one so I just left it all there. I'll open the PR in a few minutes.